server/src/main/resources/application-inmemory.conf

```diff
@@ -12,3 +12,7 @@ persistence.use-in-memory = true
 akka.persistence: { include "config/akka-persistence-inmemory.conf"}
 
 feature-flags.legacy-urls = false
+
+# This is the default configuration for the in-memory journal plugin.
```

Comment:
Has this comment been pasted in by mistake? It doesn't look relevant to the play config